cmd/main.go

```diff
@@ -95,14 +94,20 @@ func (c Configuration) logger() (logger zerolog.Logger) {
 		"off":   zerolog.PanicLevel,
 	}
 
+	// Initialize metrics, return logger without hook on error
+	metrics, err := c.metrics()
+	if err != nil {
+		logger = zerolog.New(os.Stderr).Level(lvlMap[c.Level])
+	}
```

Comment:
In order to not duplicate code, I'll advise to move these lines just before the return and use the logger we already created in if-else clauses, we need to just assign the hook to it